source/install/troubleshooting.rst

```diff
@@ -19,9 +19,40 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Take a Look at the Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This article covers accessing the numerous logs created by Mattermost applications.
+
+**Mattermost Server**
+
+1. In the *System Console* go to *Environment > Logging* and note down the path to the *File Log Directory*
```

Comment:
```suggestion
1. Navigate to **System Console > Environment > Logging** and note down the path to the **File Log Directory**.
```

Reply:
Just confirming whether these changes have been committed as I'm not able to see them on my view.

Reply:
should be solved, please confirm

Reply:
Yep, looks fine to me. :)